docs/source/reference/libraries.rst

```diff
@@ -0,0 +1,10 @@
+.. _model-libraries:
+
+Model Libraries
+===============
+
+.. automodule:: simphony.libraries.siepic
+  :members:
+
+.. automodule:: simphony.libraries.sipann
+  :members:
```

Comment:
The images for the SiEPIC library are broken.
Also, It looks like `simphony/libraries/sipann.py` is missing the pydoc string for the module.

Reply:
Not sure I get what you mean by "images are broken". I have added a pydoc string at the beginning of the SiPANN library, though.

Reply:
Ok, I had removed the images and that's why they aren't working. I just put the images back in, and I also created a new image for Halfrings, though it doesn't match the style of the other diagrams.